pkgs/development/python-modules/fastavro/default.nix

```diff
@@ -0,0 +1,51 @@
+{ buildPythonPackage
+, cython
+, dateutil
```

Comment:
Seems that this is just an alias of `python-dateutil` (which is causing the CI fail).

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/137685/commits/b60778371ad26bba37b2d3fc3474b3dbc15cb9fd.